index.html

```diff
@@ -1445,6 +1645,50 @@ <h2 id="AudioContext">
         </p>
         <dl title='interface AudioContext : BaseAudioContext' class='idl'>
           <dt>
+            Constructor
+          </dt>
+          <dd>
+            <p>
+              <span class="synchronous">When creating an <a>AudioContext</a>,
+              execute these steps:</span>
+            </p>
+            <ol>
+              <li>Set a <code>control thread state</code> to
```

Comment:
Yeah, I updated the comment.  basically you used `<em>` in other places but used `<code>` here.